Documentation/guides/app-bundles.md

```diff
@@ -0,0 +1,287 @@
+This is the Android App Bundle and `bundletool` integration
+specification for Xamarin.Android.
+
+# What are "app bundles"?
+
+[Android App Bundles][app_bundle] are a new publishing format for
+Google Play that has a wide array of benefits.
+
+* You no longer have to upload multiple APKs to Google Play:
+
+> With the Android App Bundle, you build one artifact that includes
```

Comment:
I think these block quotes should be indented 4 spaces, so that the block quote is "under" the bullet, not a peer.

e.g.

```
  * You no longer have to upload multiple APKs to Google Play:

    > With the Android App Bundle, ...

  * Dynamic Delivery...

    > Insert description here...
```

  * You no longer have to upload multiple APKs to Google Play:

    > With the Android App Bundle, ...

  * Dynamic Delivery...

    > Insert description here...

Reply:
Is the blockquoted text coming from somewhere?  If so, we should link to the source.

If not, should it be blockquoted at all?

Reply:
The quoted stuff was all from:
```
[app_bundle]: https://developer.android.com/platform/technology/app-bundle
```
👍 I'll indent it.